exercises/00-init/cypress/support/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const USER_LIST = {
```

Comment:
matter of style: i'd probably discard all CAPITALIZED_CONSTANTS as they look ugly to me, but if you prefer them, let's have them. Maybe I'm just too used to swift conventions:)

Reply:
Renamed to `userList`, see a14f4a84503240dc915dafdaa0b22f667b57b09d

I guess I was used to Python conventions :) I have no preference regarding capitalization of constants. I'm happy to adapt my style so it's more readable for others :)